app/models/activity_insight_oa_status_exporter.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class ActivityInsightOAStatusExporter < ActivityInsightExporter
+  class ExportFailed < RuntimeError; end
+
+  def initialize(file_id)
+    super()
+    @file = ActivityInsightOAFile.find(file_id)
+  end
+
+  def export
+    response = HTTParty.post webservice_url, body: to_xml(file),
+                                             headers: { 'Content-type' => 'text/xml' }, basic_auth: auth, timeout: 180
+    unless response.code == 200
```

Comment:
We should probably update `exported_oa_status_to_activity_insight` boolean to false here.  Also — now that I think about it — we should also add this field to the Rails Admin list and show pages.  That way admins can filter on it and see it in the show page.

Reply:
I added it to the rails admin pages. Question about setting `exported_oa_status_to_activity_insight` to false though- if we do that, how are we planning to handle failed exports? Anything getting exported drops out of the workflow, so if we set `exported_oa_status_to_activity_insight` as false, it won't get run through again & won't be on the radar anymore either.

Reply:
I think the flow would be something like:

- Export fails, flag is set to false, and an error is raised
- A developer notices the error, they or an admin can filter it in the admin console (or rails console for dev) to help troubleshoot
- Once the Error/bug is fixed, then the developer will have to manually re-run the export for that file

Outside of this export filter, these files have already fallen out of the workflow, so we shouldn't have to worry about anything else.